frontend/src/components/Admin/TeamEvent/TeamEventForm.tsx

```diff
@@ -18,6 +18,7 @@ const TeamEventForm = (props: Props): JSX.Element => {
   const [teamEventDate, setTeamEventDate] = useState(teamEvent?.date || '');
   const [teamEventCreditNum, setTeamEventCreditNum] = useState(teamEvent?.numCredits || '');
   const [teamEventHasHours, setTeamEventHasHours] = useState(teamEvent?.hasHours || false);
+  const [isCommunity, setIsCommunity] = useState<boolean>(false);
 
   const submitTeamEvent = () => {
     if (!teamEventName) {
```

Comment:
I think the ability to edit the community field still needs to be added. This should be something like
```
  const [isCommunity, setIsCommunity] = useStat(teamEvent?.isCommunity || false);
```
and the community field isnt included in the team event object used for the `editTeamEvent` function later on.